galley/pkg/runtime/processor.go

```diff
@@ -22,12 +22,28 @@ import (
 	"istio.io/istio/galley/pkg/metadata"
 	"istio.io/istio/galley/pkg/runtime/monitoring"
 	"istio.io/istio/galley/pkg/runtime/processing"
+	"istio.io/istio/galley/pkg/runtime/projections/serviceentry"
 	"istio.io/istio/galley/pkg/runtime/publish"
 	"istio.io/istio/galley/pkg/runtime/resource"
 	"istio.io/istio/pkg/log"
 	sn "istio.io/istio/pkg/mcp/snapshot"
 )
 
+const (
+	// Maximum wait time before deciding to publish the events.
+	serviceEntryMaxWaitDuration = 100 * time.Millisecond
```

Comment:
Can we lower this value to 10ms? Latency is important.

Reply:
Sure ... @costinm what do you think these values should be?

Reply:
@costinm @ayj @ozevren Please comment on this. We might need to try different values, the smaller the better from my perspective. Unclear if the protocol implementation can handle zero ms, but 10ms should be feasible.

Reply:
Do we expect this to be fixed in the future, or something people would tune based on their deployment? An envvar may be appropriate for initial experiments for the case. The latter case would be better exposed via a flag or as part of the settings file.